lib/js/app/queryCreator/components/Percentile/Percentile.tsx

```diff
@@ -32,16 +56,32 @@ const Percentile: FC<Props> = ({ value, onReset, onChange }) => {
   );
 
   return (
-    <>
-      <Label>{text.label}</Label>
+    <Container ref={containerRef}>
+      <AnimatePresence>
+      {
+        tooltip && (
+          <TooltipContainer
+          {...tooltipMotion}
+            initial={{ opacity: 0}}
+            animate={{
+              opacity: 1
+            }}>
+            <Tooltip mode="dark" hasArrow={false}>
+              Percentile value has to be from range 0-100
```

Comment:
Please move text copies to separate `text.json` file to keep consistency in codebase.